week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -84,7 +88,14 @@ function getTransportModes() { }
 
   Hint: Use the corresponding array method to decide if an element is member of an array.
 */
-function isAccessibleByTransportMode() { }
+function isAccessibleByTransportMode(arr, string) {
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i].includes(string)) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
Arrays methods as well here :)

Reply:
This is a bit weird way to use forEach, not wrong, but not necessary
You are receiving an array and a string and you need to check if that string is inside that array..there is a method for that